src/Elliptic/Systems/Xcts/Events/ObserveAdmIntegrals.cpp

```diff
@@ -46,9 +52,22 @@ void local_adm_integrals(
       continue;
     }
 
+    // Compute conformal factor derivatives
+    const auto deriv_conformal_factor =
+        partial_derivative(conformal_factor, mesh, inv_jacobian);
```

Comment:
Pass this in to the function as an argument. It's already being computed, e.g. for the extrinsic curvature etc, so you just have to get access to it:

1. Add `::Tags::deriv<Xcts::Tags::ConformalFactor<DataVector>, tmpl::size_t<3>, Frame::Inertial>` to your `argument_tags`.
2. Also add this tag to the list in `spacetime_quantities_compute` in `SolveXcts.hpp` to make the quantity available.